cmd/cli/install.go

```diff
@@ -301,6 +325,29 @@ func (i *installCmd) validateOptions() error {
 		return err
 	}
 
+	if i.meshConfigPath == "" {
+		// try converting to MeshConfig from osm-config configmap
+		if meshConfig, err := i.convertOsmConfigToMeshConfig(); err != nil {
+			return fmt.Errorf("Error creating MeshConfig: %s", err)
```

Comment:
wait, error? should this "try" error of it it does not succeed?

Reply:
Resolved. `install.go` won't be modified